tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -20583,7 +20583,7 @@ Curve:Biquadratic, EIRFT, 1, 0, 0, 0, 0, 0, 0, 100, 0, 100, , , Temperature, Tem
     UnitarySystems::UnitarySys::factory(*state, HVAC::UnitarySysType::Unitary_AnyCoilType, UnitarySysName, zoneEquipment, 0);
     UnitarySystems::UnitarySys *thisSys = &state->dataUnitarySystems->unitarySys[0];
     int coilIndex = CoilCoolingDX::factory(*state, "DX ClgCoil");
-    auto &this_dx_clg_coil = state->dataCoilCooingDX->coilCoolingDXs[coilIndex];
+    auto &this_dx_clg_coil = state->dataCoilCoolingDX->coilCoolingDXs[coilIndex];
```

Comment:
I'd like to think we are modeling cooing coils, like a bird's nest with birds cooing around... 

![image](https://github.com/user-attachments/assets/bc5e7da8-cef3-4919-910a-fc3d5e4f5e5d)